### ISSUE TITLE
runtime

### ISSUE THREAD

**Issue by caseykelso** [saltedcoffii/anbox-image-gapps#1]
Hi

I'm able to start the anbox container manager on ubuntu 20.04 like this

#!/bin/sh
sudo ls -l /dev/{ashmem,binder}
sudo mkdir -p /dev/binder
sudo mount -t binder binder /dev/binder
sudo ls -l /dev/{ashmem,binder}
sudo /usr/bin/anbox container-manager --data-path=/var/lib/anbox/data --android-image=/var/lib/anbox/android.img --daemon

and i'm trying to get session-manager to run, i got through the data directory error, but now it is failign saying it can't find dbus

#!/bin/sh
export SNAP_USER_COMMON=/var/lib/anbox/cache
mkdir -p /home/$USER/anbox_data
SNAP_USER_COMMON=/home/$USER/anbox_data /usr/bin/anbox session-manager 

do you have a startup script that you use?

**Comment by saltedcoffii** [comment]
No, typically I just run the anbox-container-manager with `sudo systemctl start anbox-container-manager` (you can `enable` it too), and then just run `anbox` itself. If you're using the anbox snap, I don't personally, so there's not much I can do to help, sorry.

**Comment by caseykelso** [comment]
Ah I see! I didn't realize that the deb package that this project builds has the systemd anbox services

Ok, so that works for me I'm able to start the container-manager (if I do the binder work around above)

However starting the session-manager runs into the same issue I had where the systemd service doesn't define the datapath. 

Do you start the anbox session-manager systemd service or do something manually?
